sympy/integrals/prde.py

```diff
@@ -979,6 +992,9 @@ def pad(rep):
         # 'Kmat' is the same of 'Qmat'
         Qmat = Matrix(Kmat)
 
+    V = Qmat.rowspace()
```

Comment:
The idea is to call this function for finding all rational linear relations and then choose those that satisfy some extra conditions such as one coefficient being  -1 as in the case you refer to, and possibly another being an integer. But the caller has to do that processing. `.nullspace` returns a (possibly empty) list of column matrices that are thought of as vectors. They form a basis of the nullspace. The final solutions are suitable linear combinations of the basis vectors.

(The relations can also be obtained from the rowspace, but then extra processing would be needed.)